Ruby/Week 6/OOPsie Project/modules.rb

```diff
@@ -0,0 +1,42 @@
+module Home
+  def build_home
+    number_rooms = block_given? ? yield : create_rooms
+    puts "#{@name} has successfully built a home for themselves! \nCreated #{number_rooms} rooms!"
+  end
+
+  def create_rooms
+    rooms_required = rand(3) + 2
+  end
+end
+
+module Recreation
+  def initialize(*args)
+    @playing = false
```

Comment:
Review from 1:1 - there's probably a way to initialize `@playing` without using the `initialize` method. Technically, modules can't be instantiated, so using `initialize` should be avoided.